src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -175,9 +175,99 @@ def test_analyzer_pam_logs(self, multihost, backupsssdconf):
+        5. Should succeed
+        """
+        tools = sssdTools(multihost.client[0])
+        multihost.client[0].service_sssd('stop')
+        tools.remove_sss_cache('/var/lib/sss/db/')
+        tools.remove_sss_cache('/var/log/sssd/')
+        dm_sec = ['nss', 'pam']
+        sssd_params = {'debug_level': '9'}
+        for sec_op in dm_sec:
+            tools.sssd_conf(sec_op, sssd_params, action='update')
+        multihost.client[0].service_sssd('start')
+        i_cmd = f'id foo1@{ds_instance_name}'
+        multihost.client[0].run_command(i_cmd, raiseonerr=False)
+        user = f'foo1@{ds_instance_name}'
+        client_hostname = multihost.client[0].sys_hostname
+        client = pexpect_ssh(client_hostname, user, 'Secret123',
+                             debug=False)
+        try:
+            client.login()
+        except SSHLoginException:
+            pytest.fail(f"{user} failed to login")
+        else:
+            client.logout()
+        _, stdout = analyze(multihost, 'show 1')
+        assert 'RID#' in stdout
+        _, stdout = analyze(multihost, 'show 1 --pam')
+        assert 'RID#' in stdout
+
+    def test_analyze_parse_child_logs(self, multihost, backupsssdconf):
+        """
+        :title: sssctl analyze to parse child logs from logs
+        :id: 0f009b2e-420f-40f4-ab37-e224a6607812
+        :description: sssctl analyze should able to parse child logs
+         from  logs
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2013260
+        :steps:
+        1. Configure sssd to authenticate against directory server
+        2. Enable debug_level to 9 in the 'nss', 'pam' and domain section
+        3. Restart SSSD with cleared cache
+        4. Log in as a user using ssh
+        5. Confirm child krb logs  parsed
+        :expectedresults:
+        1. Should succeed
+        2. Should succeed
+        3. Should succeed
+        4. Should succeed
+        5. Should succeed
+        """
+        tools = sssdTools(multihost.client[0])
+        multihost.client[0].service_sssd('stop')
+        tools.remove_sss_cache('/var/lib/sss/db/')
+        tools.remove_sss_cache('/var/log/sssd/')
+        dm_sec = ['nss', 'pam']
+        sssd_params = {'debug_level': '9'}
+        for sec_op in dm_sec:
+            tools.sssd_conf(sec_op, sssd_params, action='update')
+        multihost.client[0].service_sssd('start')
+        i_cmd = f'id foo1@{ds_instance_name}'
+        multihost.client[0].run_command(i_cmd, raiseonerr=False)
+        user = f'foo1@{ds_instance_name}'
+        client_hostname = multihost.client[0].sys_hostname
+        client = pexpect_ssh(client_hostname, user, 'Secret123',
+                             debug=False)
+        try:
+            client.login()
+        except SSHLoginException:
+            pytest.fail(f"{user} failed to login")
+        else:
+            client.logout()
+        _, stdout = analyze(multihost, 'show --pam --child 1')
+        assert 'child_krb5_trace' in stdout
```

Comment:
More precise assert here as well, making sure we are matching the correct user login. Matching the Krb5 error of a failed login( like bad password) could also be worthwhile, as I would not expect that error code to change.

Reply:
user name login is matched. 
Failed log in is added and assertion of failed error from child log is checked.